src/pages/AdminLogin.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {FaGoogle,FaRegEnvelope} from 'react-icons/fa'
+import {MdLockOutline} from 'react-icons/md'
+
+const AdminLogin = () => {
+  return (
+    <>
+      <div className="md:flex md:flex-col md:items-center md:justify-center w-full  min-h-screen  text-center py-2 bg-gray-100 sm:flex sm:flex-row sm:items-center sm:justify-center">
+        <div className="bg-white md:rounded-2xl md:shadow-2xl md:flex md:w-2/3  md:max-w-4xl sm:max-w-xl sm:rounded-none sm:shadow-none dark:shadow-2xl ">
+          <div className="md:w-3/5 md:p-5 sm:w-full sm:p-2 dark:bg-dark-bg dark:rounded-tl-2xl dark:rounded-bl-2xl ">
+            <div className="md:text-left md:font-bold dark:text-dark-text-fill  ">
+              <span className="text-primary"> Dev</span>pulse
+            </div>
+            <div className="py-10 sm:py-8 ">
+              <h2 className="text-2xl font-bold text-primary dark:text-dark-text-fill ">
+                Sign in to Account
```

Comment:
Here, I think it's better to say 'Sign in to your account' or just 'Sign in'

Reply:
Okay